src/main/antlr4/de/zalando/beard/BeardLexer.g4

```diff
@@ -18,6 +18,8 @@ mode INSIDE_INTERPOLATION;
 
     SLASH: '/';
 
+    COMMA: ',';
```

Comment:
As only the for is using the comma, I don't think we should include it in the lexer...